iothub/service/src/IotHubConnectionProperties.cs

```diff
@@ -54,7 +59,7 @@ public Uri BuildLinkAddress(string path)
             return builder.Uri;
         }
 
-        private static string GetIotHubName(string hostName)
+        public static string GetIotHubName(string hostName)
```

Comment:
Class is internal but this public so that unit tests can be added. This should have no impact.

Reply:
It could be internal then.